awx_collection/plugins/modules/tower_receive.py

```diff
@@ -9,91 +9,23 @@
 
 
 ANSIBLE_METADATA = {'metadata_version': '1.1',
-                    'status': ['preview'],
+                    'status': ['deprecated'],
                     'supported_by': 'community'}
 
 
 DOCUMENTATION = '''
 ---
 module: tower_receive
+deprecated:
+  removed_in: "3.7"
+  why: Deprecated in favor of upcoming C(_export) module.
+  alternative: Once published, use M(tower_export) instead.
 author: "John Westcott IV (@john-westcott-iv)"
 version_added: "2.8"
 short_description: Receive assets from Ansible Tower.
 description:
     - Receive assets from Ansible Tower. See
       U(https://www.ansible.com/tower) for an overview.
-options:
-    all:
-      description:
-        - Export all assets
-      type: bool
-      default: 'False'
-    organization:
-      description:
-        - List of organization names to export
-      default: []
-      type: list
-      elements: str
-    user:
-      description:
-        - List of user names to export
-      default: []
-      type: list
-      elements: str
-    team:
-      description:
-        - List of team names to export
-      default: []
-      type: list
-      elements: str
-    credential_type:
-      description:
-        - List of credential type names to export
-      default: []
-      type: list
-      elements: str
-    credential:
-      description:
-        - List of credential names to export
-      default: []
-      type: list
-      elements: str
-    notification_template:
-      description:
-        - List of notification template names to export
-      default: []
-      type: list
-      elements: str
-    inventory_script:
-      description:
-        - List of inventory script names to export
-      default: []
-      type: list
-      elements: str
-    inventory:
-      description:
-        - List of inventory names to export
-      default: []
-      type: list
-      elements: str
-    project:
-      description:
-        - List of project names to export
-      default: []
-      type: list
-      elements: str
-    job_template:
-      description:
-        - List of job template names to export
-      default: []
-      type: list
-      elements: str
-    workflow:
-      description:
-        - List of workflow names to export
-      default: []
-      type: list
-      elements: str
```

Comment:
I'm not at all clear on what happened with the options.

Reply:
I incorrectly thought I had to remove those after adding the deprecation info in the descriptions.  Will revert this.